rtcicetransport.go

```diff
@@ -3,4 +3,28 @@ package webrtc
 // RTCIceTransport allows an application access to information about the ICE
 // transport over which packets are sent and received.
 type RTCIceTransport struct {
```

Comment:
Can we bring these back?

Reply:
Oh these never existed!!! This is all the new stuff that I am building. Again, it the PR just because I pushed stuff in order to fix master. This will be created in the next few PRs.